app/controllers/api/serials_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::SerialsController < Api::ApplicationController
+  def index
+    serials = current_site.serials.includes(:posts).order(id: :desc).page(params.dig(:page, :number)).per(params.dig(:page, :size))
+
+    render json: serials, include: "post"
```

Comment:
`include: "post"` will be ignored.
If you need, you should use `include: "posts"`.